tauolapp.spec

```diff
@@ -1,13 +1,13 @@
 ### RPM external tauolapp 1.1.5
-Source: http://service-spi.web.cern.ch/service-spi/external/MCGenerators/distribution/tauola++/tauola++-%{realversion}-src.tgz
+Source: http://tauolapp.web.cern.ch/tauolapp/resources/TAUOLA.1.1.5/TAUOLA.%{realversion}-LHC.tar.gz
```

Comment:
@davidlange6 ,please use `TAUOLA.%{realversion}` instead of `TAUOLA.1.1.5`

Reply:
whoops... thanks for noticing that